rocAL/rocAL_pybind/amd/rocal/plugin/pytorch.py

```diff
@@ -1,7 +1,7 @@
 import torch
 import numpy as np
 import rali_pybind as b
```

Comment:
remove empty file above __init__.y

Reply:
The __init__.py files are required to make Python treat the directories as containing packages
In the simple case, they can also be empty.
[__init__.py](http://docs.python.org/tutorial/modules.html#packages):